### PR TITLE
Remove `ARM_MUSCA_A1` support

### LINKED ISSUE #28: Remove MUSCA A1 support
Mbed OS has removed MUSCA A1, so we should drop it from this test repo as well.

### PR DESCRIPTION
**Pull request by jainvikas8**
Mbed OS 6.0 has removed support for ARM_MUSCA_A1 target, therefore the
support needs to be dropped for PSA Compliance tests.

Fixes #28

### CODE REVIEW

**Comment by Patater** [comment]
Please update `tfm_ns_import.yaml` as well.

**Comment by jainvikas8** [comment]
Forced pushed to address review comments from @Patater

File: tfm_ns_import.yaml [hunk 1]
```diff
@@ -152,16 +146,6 @@
...
             }
          ],
-         "ARM_MUSCA_A1": [
-            {
-                "src": "platform/ext/target/musca_a/Device/Include/platform_base_address.h",
-                "dst": "tfm/targets/TARGET_ARM_SSG/TARGET_MUSCA_A1/device/platform_base_address.h"
-            },
-            {
-                "src": "platform/ext/target/musca_a/tfm_peripherals_def.h",
-                "dst": "tfm/targets/TARGET_ARM_SSG/TARGET_MUSCA_A1/device/tfm_peripherals_def.h"
```
Comment (Patater) [inline tfm_ns_import.yaml:162]: These destination folders (inside this regression test repo) are still present and should be removed as part of this commit.

`./tfm/targets/TARGET_ARM_SSG/TARGET_MUSCA_A1`

**Comment by jainvikas8** [comment]
Forced pushed to address review comments from @Patater

**Pull request merged**